Up2dateService/Up2dateClient/Client.cs

```diff
@@ -141,6 +141,22 @@ private void OnDeploymentAction(IntPtr artifact, DeploymentInfo info, out Client
             setupManager.OnDownloadFinished(info.artifactFileName);
 
             WriteLogEntry("download completed.", info);
+
+
+            if (info.updateType == "skip")
+            {
+                result.Message = "skip installation - not requested";
+                WriteLogEntry(result.Message, info);
+                return;
+            }
+
+            if (setupManager.IsPackageInstalled(info.artifactFileName))
```

Comment:
it looks dangerous - setupManager cannot properly define if the package was installed with the same revision. Consider to remove

Reply:
Removed in [5b04ed4](https://github.com/rtsoft-gmbh/up2date-win/pull/24/commits/5b04ed46c8a08391943728d6060c3e2842be693c)